docs/pages/attributesJSON.rst

```diff
@@ -502,12 +509,37 @@ For example, :ref:`ObjectAttributes.get_user_config` returns the configuration o
                 "can open"
             ],
             "custom_object_properties":{
-                "is_gripped": "false",
+                "is_gripped": false,
                 "temperature": 10.0,
             },
         }
     }
 
+The attributes parser interprets the type of the data in the user-defined json fields based on each field's data and layout, with a few exceptions, as illustrated below:
+
+User-Defined JSON Field type mappings
+-------------------------------------
+
+.. class:: m-table m-fullwidth
+
+======================= =========================== ===================
+JSON field data example Habitat-Sim internal type   Notes
+======================= =========================== ===================
+10.0                    double
+7                       integer
+"can grip"              string
+false                   boolean
+[0,1]                   Magnum Vector2 (float)
+[0,1,2]                 Magnum Vector3 (float)
+[0,1,3,4]               Magnum Vector4 (float)      1
+[0,1,2,3]               Magnum Quaternion (float)   1
+[1,2,3,4,5,6,7,8,9]     Magnum Matrix3 (float)      2
+
+======================= =========================== ===================
+
+1 - If a length-4 numeric vector's tag contains as a substring 'quat', 'orient' or 'rotat', case-insensitive, the object will be loaded and processed as a Magnum::Quaternion (w,x,y,z) by the parser. Otherwise, it will be loaded as a Magnum::Vector4.
+2 - A length-9 numeric vector will be mapped into a Magnum Matrix3 in column-major order.
```

Comment:
I think you need an extra newline here to break the paragraph.

Reply:
Wasn't sure if the doc parser would do that for us.

Reply:
Yeah, I used the github rst file viewer to noticed this. Maybe different parsers will do things differently, but I think generally a newline is required in rst to break paragraphs.